### PR TITLE
LPD-42678

### PR DESCRIPTION
**Pull request by brianikim**
@luca-pellizzon ,

https://liferay.atlassian.net/browse/LPD-42678.

The exception implementation follows pattern from previous ticket [LPD-38934](https://github.com/brianchandotcom/liferay-portal/pull/156714) which was requested by BChan.

Sincerely,
Brian I. Kim

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/util/CommerceOrderItemQuantityFormatterImpl.java [hunk 1]
```diff
@@ -148,6 +202,80 @@ private DecimalFormat _getDecimalFormat(
...
+	private void _validateQuantity(String className, String quantity)
+		throws Exception {
+
+		if (Validator.isNull(className) || _hasCommaDecimalPattern(quantity) ||
+			_hasPeriodDecimalPattern(quantity)) {
+
+			return;
+		}
+
+		if (Objects.equals(
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/util/CommerceOrderItemQuantityFormatterImpl.java:226]: This is not really flexible... what if we add something else with quantities in the future?
I guess it is ok for now, but having a registry of "quantityFormatter" components identified by the className will allow us to be more flexible for extensions

**Review by luca-pellizzon — APPROVED** [review]

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:forward:force

**Pull request closed without merging**